### ISSUE TITLE
No parser was explicitly specified

### ISSUE THREAD

**Issue by javell** [richardasaurus/wiki-api#18]
I get this message and it says to change line 9 in run.py, but there is no such file

**Comment by richardasaurus** [comment]
Can you post the exact output you're getting?

**Comment by abuvanth** [comment]
reproduce this issue

 BeautifulSoup(YOUR_MARKUP})

to this:

 BeautifulSoup(YOUR_MARKUP, "lxml")

**Comment by happynear** [comment]
Same problem.
```
C:\ProgramData\Anaconda3\lib\site-packages\bs4\__init__.py:181: 
UserWarning: No parser was explicitly specified, so I'm using the best available HTML parser for this system ("lxml"). 
This usually isn't a problem, but if you run this code on another system, or in a different virtual environment, it may use a different parser and behave differently.
The code that caused this warning is on line 1683 of the file C:\Program Files\JetBrains\PyCharm Community Edition 2017.3\helpers\pydev\pydevd.py. To get rid of this warning, change code that looks like this:
 BeautifulSoup([your markup])
to this:
 BeautifulSoup([your markup], "lxml")
  markup_type=markup_type))
```

**Comment by richardasaurus** [comment]
fixed in `1.2.6`
